node/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = '2021'
 license = 'Apache-2.0'
 repository = 'https://github.com/digicatapult/dscp-node/'
 name = 'dscp-node'
-version = '4.0.0'
```

Comment:
need to also update the version of the dependency `dscp-node-runtime` in this file and then regenerate the cargo lock